rust/operator-binary/src/airflow_controller.rs

```diff
@@ -602,68 +634,15 @@ async fn build_executor_template(
     Ok(())
 }
 
-/// The server-role service is the primary endpoint that should be used by clients that do not perform internal load balancing,
-/// including targets outside the cluster.
-fn build_role_service(
```

Comment:
Do we still want to keep the old service around for a cycle or so to ease migration? Not sure.